tests/featurizers/test_featurizers.py

```diff
@@ -93,7 +93,7 @@ def test_count_featurizer(tmp_path: pathlib.Path):
 
     labeler = CodeLabeler([femr_outcome_code], time_horizon, [femr_admission_code])
 
-    patient: femr.Patient = cast(femr.Patient, database[0])
+    patient: femr.Patient = cast(femr.Patient, database[next(iter(database))])
```

Comment:
If this is a common pattern - maybe implement a `database.next()` helper function for users

Reply:
This is not a common pattern. Pulling the first patient from a patient database is sorta weird.